src/components/tl/ucp/reduce_scatter/reduce_scatter_ring.c

```diff
@@ -129,8 +134,11 @@ static void ucc_tl_ucp_reduce_scatter_ring_progress(ucc_coll_task_t *coll_task)
         reduce_target = s_scratch[id];
         step          = task->tagged.send_posted;
         prevblock     = (rank - 1 - step + size) % size;
-        prevblock =
+        prevblock     =
             ucc_ep_map_eval(task->reduce_scatter_ring.inv_map, prevblock);
+        if (team->cfg.use_reordering) {
```

Comment:
we could use the inline form here
```
prevblock = ucc_ep_map_eval(team->cfg.use_reordering ?  task->reduce_scatter_ring.inv_map : task->subset.map, prevblock) 
```

Reply:
no, it is not one or the other, if reordering is being used then we want both mappings to happen one after the other.